components/myServices/ServiceIProvide.tsx

```diff
@@ -0,0 +1,21 @@
+import ServiceCard from "@/components/myServices/ServiceCard";
+
+const ServiceIProvide = () => {
+  return (
+    <section className="flex flex-col items-center gap-10 bg-white900 py-12 dark:bg-black300 sm:bg-white800 xl:px-[5.31rem] xl:pb-[8.9375rem] xl:pt-[4.5rem]">
+      <h1 className="mySkills relative dark:text-white900 xl:text-[3rem] xl:tracking-[-0.03rem]">
```

Comment:
Why are you using a className called "mySkills" inside "ServiceIProvide?" Maybe you can give it a new name, that is more in aligned with it being used in more than one place?

Reply:
My mistake! Thanks.